### PR TITLE
Update tests, reoginise for docs

### PR DESCRIPTION
**Pull request by PiVortex**
- Updated tests for common format 
- Updated test for JS part 3 to be one test
- Changed some code around for ease of writing docs

### CODE REVIEW

File: contract-ts/03-owner-claims-winner-gets-nft/src/contract.ts
Comment (PiVortex) [reply contract-ts/03-owner-claims-winner-gets-nft/src/contract.ts:68]: Many NFT and FT contracts have a check that you cannot transfer to yourself including the [NFT contract we are using](https://github.com/near-examples/nft-tutorial/blob/main/nft-contract-approval/src/internal.rs#L169). Is it still ok because this is a cross contract call and if it fails who cares?

Comment (gagdiez) [reply contract-ts/03-owner-claims-winner-gets-nft/src/contract.ts:68]: Indeed

**Review by gagdiez — APPROVED** [review]

**Comment by gagdiez** [comment]
@race-of-sloths I invoke you

**Comment by race-of-sloths** [comment]
### 🙁 This repository is not a part of the [Race of Sloths](https://race-of-sloths.com)
Please ask the maintainer to [contact us](https://github.com/NEAR-DevHub/race-of-sloths/issues/new/choose) to join the race!

<details><summary><strong>What is the Race of Sloths</strong></summary>
<p>

Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag us inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/PiVortex)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!

</p>
</details>

**Pull request merged**
